### DISCUSSION TITLE
What do B, L, H, g, and G stand for in the ViT model names?

### DISCUSSION THREAD

**Discussion by thesofakillers** [mlfoundations/open_clip#489] [Q&A] [answered]
Based on the ViT paper, I'm pretty sure that B, L and H stand for base, large and huge (model size). 

What do g and G stand for? BingChat said giant and Grand but failed to provide a reference.

Thanks!

**Comment by mitchellnw** [comment, marked as answer]
G = giant, lower and upper case are small and big giant

**Comment by thesofakillers** [comment]
Thx!
